content/en/docs/reference/multi-cluster.md

```diff
@@ -8,6 +8,7 @@ lastmod: 2018-01-02
 categories: [getting started]
 keywords: [install,kubernetes]
 weight: 30
+draft: true
```

Comment:
I don't think the whole page should become a draft. It is just the `jxl` part which should be removed.

Unpublishing this page causes these link errors:

```
- ./tmp-website/community/office_hours/2019-04-19/index.html
  *  internally linking to /docs/reference/multi-cluster/, which does not exist (line 0)
     <a href="/docs/reference/multi-cluster/">multi cluster</a>
  *  internally linking to /docs/reference/multi-cluster/, which does not exist (line 0)
     <a href="/docs/reference/multi-cluster/">Multi-cluster</a>
htmlproofer 3.15.3 | Error:  HTML-Proofer found 2 failures!
```

Reply:
That's fair. +1 on just removing the JXL bit. @sharepointoscar @vfarcic

Reply:
True true, let me edit that portion and publish an update!